Tools/TestWebKitAPI/Tests/WebKitCocoa/iOSMouseSupport.mm

```diff
@@ -237,12 +237,14 @@ - (UIPointerInteraction *)pointerInteraction
         m_gestureButtonMaskSwizzler = makeUnique<InstanceMethodSwizzler>(UIGestureRecognizer.class, @selector(buttonMask), imp_implementationWithBlock([&](UIGestureRecognizer *) {
             return m_buttonMask;
         }));
+        m_unusedEvent = adoptNS([UIEvent new]);
     }
 
     MouseEventTestHarness() = delete;
 
     void mouseMove(CGFloat x, CGFloat y)
     {
+        // FIXME: This test helper should handle mouse drags.
```

Comment:
Can you write a bug for this FIXME?

Reply:
Sure — https://bugs.webkit.org/show_bug.cgi?id=262757